src/test/java/system/UnitTesting/CardEffectUnitTesting.java

```diff
@@ -10,13 +11,27 @@
 import java.util.List;
 
 class CardEffectUnitTesting {
+
+    private GameState gameState;
+    private User user;
+    private DrawDeck drawDeck;
+    private GameManager gameManager;
+    private List<User> users;
+    private List<Card> cards;
+
+    @BeforeEach
+    void testMockCreation() {
```

Comment:
```suggestion
    void setUp() {
```
Normally test methods start with `test` but this is not a test method